caffeine@patapon.info/preferences/displayPage.js

```diff
@@ -90,23 +92,13 @@ class CaffeineDisplayPage extends Adw.PreferencesPage {
                 page_size: 0,
                 value: this._settings.get_int(this._settingsKey.INDICATOR_POSITION)
             }),
-            climb_rate: 1,
-            digits: 0,
-            numeric: true,
-            valign: Gtk.Align.CENTER
-        });
-        let posIndicatorOffsetRow = new Adw.ActionRow({
-            title: _('Status indicator position'),
-            subtitle: _('The position relative of indicator icon to other items'),
-            activatable_widget: this.posIndicatorOffsetButton
-        });
-        posIndicatorOffsetRow.add_suffix(this.posIndicatorOffsetButton);
+        })
```

Comment:
```suggestion
        });
```

Missing a semi-colon

Reply:
It's fixed now. Thank you